api/src/main/resources/messages.properties

```diff
@@ -96,7 +96,9 @@ patientflags.errors.flagNameExists=A flag with this name already exists
 patientflags.errors.tagNameExists=A tag with this name already exists
 patientflags.errors.noPriority=Priority is required
 patientflags.errors.noPriorityName=Priority name is required
+patientflags.errors.noStyleName=Priority style is required

```

Comment:
@druchniewicz did you intend to append the string `^M` on the property?

Reply:
no, it was not intended, it is related to different end-line characters on different systems and I haven't seen it during development. I already fixed it